pylint/checkers/stdlib.py

```diff
@@ -147,6 +150,13 @@ class StdlibChecker(BaseChecker):
             "trivial! Minimize the number of libraries you call into."
             "https://docs.python.org/3/library/subprocess.html#popen-constructor",
         ),
+        "W1510": (
+            "Using subprocess.run without explicitly set `check` is not recommended.",
+            "subprocess-run-check",
+            "The check parameter should always be used with explicitly set "
+            "`check` keyword to make clear what the error-handling behavior is."
+            "https://docs.python.org/3/library/subprocess.html#subprocess.runs",
```

Comment:
The url has an extra `s` at the end.
https://docs.python.org/3/library/subprocess.html#subprocess.runs